docker/ubuntu-22.04-amd64-openssl-3.0.x/Dockerfile

```diff
@@ -0,0 +1,52 @@
+ARG UBUNTU_VERSION="22.04"
+
+FROM ubuntu:${UBUNTU_VERSION}
+
+ARG NODE_VERSION="18.12.1"
```

Comment:
We discussed that this could use the most recent LTS version and that we could have a test for the latest version too

Reply:
Easy way would be to replace with https://github.com/nodesource/distributions


Using Ubuntu

curl -fsSL https://deb.nodesource.com/setup_lts.x | sudo -E bash - &&\
sudo apt-get install -y nodejs

Using Debian, as root

curl -fsSL https://deb.nodesource.com/setup_lts.x | bash - &&\
apt-get install -y nodejs

Node.js Current (v19.x):
Using Ubuntu

curl -fsSL https://deb.nodesource.com/setup_current.x | sudo -E bash - &&\
sudo apt-get install -y nodejs

Using Debian, as root

curl -fsSL https://deb.nodesource.com/setup_current.x | bash - &&\
apt-get install -y nodejs